scripts/repeatIT.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+targetTest=${1:-ExpositionIT}
+
+logFile=/tmp/mvn-${targetTest}-$(date +"%Y%m%dT%H%M%S").log
+
+mvn clean install -DskipTests=true
+
+for COUNT in {1..10} ; do
+    echo "=============== RUN ${COUNT} STARTING =============== " | tee -a "${logFile}"
+    mvn verify -Dit.test="${targetTest}" -pl :integrationtests |& "${logFile}"
```

Comment:
When running this version (with `|&`) I get a `line 16: /tmp/mvn-ExpositionIT-03-08-2023_10:08.log: Permission denied`, which is odd. When running with `>>` I don't get this error.

Reply:
Based on [this](https://stackoverflow.com/questions/35384999/what-does-mean-in-bash#comment135016571_35385038) it seems it may be an issue of support for the `|&` operator.

Reply:
> I think it's probably interpreting the tmp file as a command (because it pipes stdout from the first command into stdout for the second command) and then complaining it has no execute permissions on the file

Hmm that sounds right. Sigh I guess I need to add `>> ${logFile}` so it the pipe appended to the file...

Reply:
You could also possibly leave it largely as is and use `tee`, i.e.
```suggestion
    mvn verify -Dit.test="${targetTest}" -pl :integrationtests |& tee -a "${logFile}"
```
or
```suggestion
    mvn verify -Dit.test="${targetTest}" -pl :integrationtests 2>&1 | tee -a "${logFile}"
```

Reply:
fixed in https://github.com/kroxylicious/kroxylicious/pull/489/commits/d3e99462cb7cb5356081a712a740a58ea3f17a16

Reply:
Yeah, I consider `tee`, but didn't really want to dump the full output to the terminal.